libs/langchain-community/src/chat_models/zhipuai.ts

```diff
@@ -0,0 +1,444 @@
+import { BaseChatModel, type BaseChatModelParams } from "@langchain/core/language_models/chat_models";
+import { AIMessage, type BaseMessage, ChatMessage } from "@langchain/core/messages";
+import { type ChatResult } from "@langchain/core/outputs";
+import { type CallbackManagerForLLMRun } from "@langchain/core/callbacks/manager";
+import { getEnvironmentVariable } from "@langchain/core/utils/env";
+// eslint-disable-next-line import/no-extraneous-dependencies
+import * as jwt from 'jsonwebtoken';
```

Comment:
Can we add this as a peer dependency instead?

https://github.com/langchain-ai/langchainjs/blob/main/.github/contributing/INTEGRATIONS.md#third-party-dependencies

Reply:
> Can we add this as an optional peer dependency instead?
> 
> https://github.com/langchain-ai/langchainjs/blob/main/.github/contributing/INTEGRATIONS.md#third-party-dependencies

yes, thanks for checking, please help to check this again